RelVal/ReleaseValidation.C

```diff
@@ -9,28 +9,30 @@ TFile *fileSummaryOutput = nullptr;
 TFile *fileTestSummary = nullptr;
 
 TString prefix = "";
-int correlationCase = 0; // at the moment...Later I need to check that
+int correlationCase = 0; // at the moment I assume no error collation ... Later I need to check that
 
 struct results
 {
   bool a;
   double b;
   double c;
-  bool d;
+  double d;
+  bool e;
 };
 
-// define "a monte" the possible avaiable tests
+// define "a monte" the possible available tests
 enum options {
   Chi2    = 0x01,
   BinCont     = 0x02,
-  Nentries   = 0x04,
+  BinContInt   = 0x04, //BinContInt is similar to BinCont, but the bin content is normalized by the integral 
   // ...
 };
 
 
-void ProcessFile(TString fname, TString dirToAnalyse); // needs to be declared
+void ProcessFile(TString fname, TString dirToAnalyse); 
```

Comment:
of course you can use `TString` here. In case it's only accessed to be read and not modified, I would make it `const&`